tests/system/feature_store/test_feature_store.py

```diff
@@ -592,6 +592,36 @@ def test_nosql_no_path(self):
             1
         ].get_path().get_absolute_path() == fset.get_target_path("nosql")
 
+    @pytest.mark.parametrize("local", [True, False])
+    def test_ingest_with_format_run_project(self, local):
+        first_data = pd.read_csv("/Users/Tomer_Mamia/Downloads/first_data.csv")
```

Comment:
You need to replace this with a file that will exist for system tests. Either add it as a file in the test folder, or create it when running the test.